compiler/java/src/main/java/com/c/lexer/TokenType.java

```diff
@@ -0,0 +1,37 @@
+package com.c.lexer;
+
+public enum TokenType {
+    // End of file
+    EOF,
+
+    // Keywords
+    IF,
+    HAS,
+    NOT,
+
+    // Boolean values
+    TRUE,
+    FALSE,
+
+    // Operators
+    OR, // |
+    EQUALS, // ==
+    NOT_EQUALS, // !=
```

Comment:
Do you want to implement the new operands directly or do you want to postpone that?

Reply:
will implement them directly